10-angular/custom_directives/readme.md

```diff
@@ -327,8 +332,9 @@ We do so using attributes:
 
 ```js
 (function(){
-  var directives = angular.module('grumbleDirectives',[]);
-  directives.directive('myCustomDirective', function(){
+  angular
```

Comment:
line ~328
` <div class="grumbles" ng-repeat="grumble in grumblesCtrl.grumbles">`
needs to reference `GrumbleIndexViewModel` instead of `grumblesCtrl`